cocotb/share/makefiles/Makefile.sim

```diff
@@ -79,7 +79,19 @@ ifeq ($(MAKECMDGOALS),help)
     help_vars := $(subst %,${newline},$(shell cocotb-config --help-vars | tr \\n %))
     $(info ${help_vars})
     # is there a cleaner way to exit here?
-    $(error "Stopping after printing help")
+    $(error Stopping after printing help)
+endif
+
+# Safety net to bail out early if users go into this Makefile without making
+# cocotb-config available in their PATH.
+# Use override to prevent users from overriding COCOTB_CONFIG_CMD, which is not
+# supported. (Modify PATH instead.)
+override COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
```

Comment:
Would not be better to define this in `Makfile.inc` ? One can just include `Makefie.icarus` now and will never see `Makefile.sim`